lib/find-dupes.js

```diff
@@ -2,7 +2,15 @@
 const dedupe = require('./dedupe.js')
 const usageUtil = require('./utils/usage.js')
 
-const usage = usageUtil('find-dupes', 'npm find-dupes')
-const cmd = (args, cb) => dedupe({ dryRun: true }, cb)
+class FindDupes {
+  /* istanbul ignore next - see test/lib/load-all-commands.js */
+  get usage () {
+    return usageUtil('find-dupes', 'npm find-dupes')
+  }
 
-module.exports = Object.assign(cmd, { usage })
+  exec (args, cb) {
+    // TODO this should really be this.npm.commands.dedupe
+    dedupe({ dryRun: true }, cb)
```

Comment:
this TODO is pb already unlocked by #2802

Reply:
It's been fixed in this PR cause it was a bug